geane/CMakeLists.txt

```diff
@@ -27,8 +27,8 @@ ${CMAKE_SOURCE_DIR}/alignment
 
 Set(SYSTEM_INCLUDE_DIRECTORIES
   ${ROOT_INCLUDE_DIR} 
-  ${GEANT3_INCLUDE_DIR}
   ${Geant3_INCLUDE_DIRS}
+  ${Geant3_INCLUDE_DIRSS}
```

Comment:
Do we really have a variable Geant3_INCLUDE_DIRSS or is this a typo?

Reply:
Typo